packages/autocomplete-css/update/update.js

```diff
@@ -103,8 +104,13 @@ async function update(params) {
     pseudoSelectors: pseudoSelectors
   };
 
+  // Now lets sort the properties according to length
+  completions.properties = lengthSort(completions.properties);
+  // Then lets sort properties by popularity
+  completions.properties = await popularSort(completions.properties);
```

Comment:
Here's a magic trick that makes the comments superfluous ;-)

```suggestion
  completions.properties = sortByLength(completions.properties);
  completions.properties = await sortByPopularity(completions.properties);
```

Reply:
Fantastic point, lemme make that change real quick